pom.xml

```diff
@@ -5,12 +5,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.14</version>
+    <version>2.17</version>
   </parent>
   
   <properties>
-    <jenkins.version>2.23-SNAPSHOT</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.26-SNAPSHOT</jenkins.version>
```

Comment:
Did not care to use a patched core in my case. But, 2.19.1 suffered from [JENKINS-35184](https://issues.jenkins-ci.org/browse/JENKINS-35184), making it impossible to connect an agent. Was fixed as of 2.24, I just happened to have 2.26-SNAPSHOT locally.